scripts/website/expand_templates/exclusions.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/golangci/golangci-lint/pkg/config"
+)
+
+func getDefaultExclusions() string {
+	defaultExcludePatterns, err := readJSONFile[[]config.ExcludePattern](filepath.Join("assets", "default-exclusions.json"))
+	if err != nil {
+		panic(err)
```

Comment:
Maybe it would be better for `getDefaultExclusions()` to return an error instead of causing a panic? After all, `getDefaultExclusions()` is invoked within `buildTemplateContext()`, which is designed to handle errors.